packages/carbon-component-mapper/src/files/date-picker.js

```diff
@@ -24,13 +24,15 @@ const DatePicker = (props) => {
 
 DatePicker.propTypes = {
   isDisabled: PropTypes.bool,
+  isRequired: PropTypes.bool,
   datePickerType: PropTypes.string,
   DatePickerProps: PropTypes.object,
   WrapperProps: PropTypes.object
 };
 
 DatePicker.defaultProps = {
-  datePickerType: 'single'
+  datePickerType: 'single',
+  isRequired: false
```

Comment:
We don't need to set up `false` as a default value. By default, `undefined` is set so it works the same.